maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/NullStatisticsReporter.java

```diff
@@ -35,7 +36,7 @@
 
     public NullStatisticsReporter()
     {
-        super( FileUtils.getTempDirectory() );
+        super( FileUtils.getFile( FileUtils.getTempDirectory(), RandomStringUtils.randomAlphabetic( 24 ) ) );
```

Comment:
It would be nice to have protected no-arg constructor of `StatisticsReporter` for these purposes and null variables of `File` and `RunEntryStatisticsMap`, etc. Not necessary to create new objects if unused.